### ISSUE TITLE
Help wanted! Feather SVG-sprite

### ISSUE THREAD

**Issue by Hinigatsu** [feathericons/feather#751]
As explained in the README: [https://github.com/feathericons/feather#svg-sprite](https://github.com/feathericons/feather#svg-sprite)

You simply need to serve this

**CSS**:

    .feather {
        width: 24px;
        height: 24px;
        stroke: currentColor;
        stroke-width: 2;
        stroke-linecap: round;
        stroke-linejoin: round;
        fill: none;
    }

**HTML**:

    <svg class="feather">
        <use xlink:href="path/to/dist/feather-sprite.svg#circle"/>
    </svg>

___

But, when trying this:

```
<!DOCTYPE html>
<head>
	<title>Feather Test</title>

	<style>
	.feather {
		width: 24px;
		height: 24px;
		stroke: currentColor;
		stroke-width: 2;
		stroke-linecap: round;
		stroke-linejoin: round;
		fill: none;
	}
	</style>
</head>
<body>
	<!-- Try1 -->
	<svg class="feather">
		<use xlink:href="https://unpkg.com/feather-icons@4.24.1/dist/feather-sprite.svg#circle" />
	</svg>

	<!-- Try2 -->
	<svg class="feather">
		<use xlink:href="https://unpkg.com/feather-icons/dist/feather-sprite.svg#circle" />
	</svg>

	<!-- Try3 -->
	<!-- feather-sprite in the same directory as this .html file -->
	<svg class="feather">
		<use xlink:href="feather-sprite.svg#circle" />
	</svg>
</body>
</html>
```

    

I only get an empty page. 

All links properly refers to the file needed.

What I'm missing?!

**Comment by cryptic22** [comment]
You are trying to get SVG use method for loading SVG from an external source which is not possible due to security issue. Refer to this: https://bugs.chromium.org/p/chromium/issues/detail?id=470601.
Solution: You will have to install feather icons in your local and serve it from your dist folder.

**Comment by locness3** [comment]
> Solution: You will have to install feather icons in your local and serve it from your dist folder.

Isn't that what he have done on last try ?

**Comment by cryptic22** [comment]
> > Solution: You will have to install feather icons in your local and serve it from your dist folder.
> 
> Isn't that what he has done on last try ?

I tried on codesandbox. It is working properly. Here is the link: https://codesandbox.io/s/feather-icon-mdihq

**Comment by Hinigatsu** [comment]
My goal with SVG Sprite is simply to avoid using scripts for icons...

As said, I already tried serving a local copy of feather-sprite.svg, but didn't had success. I also downloaded a copy of @cryptic22 code and still getting a blank page.  
So I'm starting to think it's a local issue, since every tutorial about SVG Sprite I could find uses the same method on feather/README...

I'll continuing trying to find a way to serve things as an sprite (or even trough a CDN), but for now, I simply think that downloading from [feathericons.com](https://feathericons.com/) and referring them as an `<img>` element is the way!

Still, I don't get why the `<svg><use href="path" /></use>` doesn't work :/
